sql-plugin/src/main/330+-nondb/scala/com/nvidia/spark/rapids/shims/Spark330PlusShims.scala

```diff
@@ -232,7 +167,7 @@ trait Spark330PlusShims extends Spark321PlusShims with Spark320PlusNonDBShims {
             GpuDivideDTInterval(lhs, rhs)
         })
     ).map(r => (r.getClassFor.asSubclass(classOf[Expression]), r)).toMap
-    super.getExprs ++ map
+    super.getExprs ++ map ++ roundingExprs
```

Comment:
if you make it an object this trait won't need to change inheritance and would call `RondingShims.roundingExprs`